src/presentation/controller/registration/Routing.ts

```diff
@@ -65,21 +65,21 @@ const registrationRoutingGeneralPartnerChoice = {
 const registrationRoutingAddGeneralPartnerPerson = {
   previousUrl: url.GENERAL_PARTNER_CHOICE_URL,
   currentUrl: url.ADD_GENERAL_PARTNER_PERSON_URL,
-  nextUrl: POSTCODE_USUAL_RESIDENTIAL_ADDRESS_URL, // will be changed to Where is the usual residential address (not ready)
+  nextUrl: GENERAL_PARTNER_USUAL_RESIDENTIAL_ADDRESS_CHOOSE_TERRITORY_URL,
   pageType: RegistrationPageType.addGeneralPartnerPerson
 };
 
 const registrationRoutingAddGeneralPartnerLegalEntity = {
   previousUrl: url.GENERAL_PARTNER_CHOICE_URL,
   currentUrl: url.ADD_GENERAL_PARTNER_LEGAL_ENTITY_URL,
-  nextUrl: POSTCODE_USUAL_RESIDENTIAL_ADDRESS_URL, // will be changed to Where is the usual residential address (not ready)
+  nextUrl: GENERAL_PARTNER_USUAL_RESIDENTIAL_ADDRESS_CHOOSE_TERRITORY_URL,
```

Comment:
the nextUrl for the legal entity shouldn't go to the URA page, it should go to LIMITED_PARTNERS_URL for now.